docs/documentation/platform/gateways/relay-deployment.mdx

```diff
@@ -0,0 +1,424 @@
+   # Example iptables rule
+   iptables -A INPUT -p tcp --dport 2222 -j ACCEPT
+   ```
+
+2. **TCP with TLS on port 8443** - For platform connections
+
+   ```bash
+   # Example iptables rule
+   iptables -A INPUT -p tcp --dport 8443 -j ACCEPT
+   ```
+
+3. **HTTPS to Infisical API** - For certificate requests and API communication
+   ```bash
+   # Example iptables rule
+   iptables -A OUTPUT -p tcp --dport 443 -d app.infisical.com -j ACCEPT
+   ```
+
+## Cloud Provider Configuration
+
+### AWS EC2
+
+Configure security groups:
+
+```json
+{
+  "SecurityGroupRules": [
+    {
+      "IpProtocol": "tcp",
+      "FromPort": 2222,
+      "ToPort": 2222,
+      "CidrIpv4": "0.0.0.0/0",
+      "Description": "SSH for gateway connections"
+    },
+    {
+      "IpProtocol": "tcp",
+      "FromPort": 8443,
+      "ToPort": 8443,
+      "CidrIpv4": "0.0.0.0/0",
+      "Description": "TCP with TLS for platform connections"
+    }
+  ]
+}
+```
+
+### Google Cloud Platform
+
+Configure firewall rules:
+
+```bash
+# Allow SSH from gateways
+gcloud compute firewall-rules create allow-gateway-ssh \
+  --allow tcp:2222 \
+  --source-ranges 0.0.0.0/0 \
+  --description "Allow SSH from gateways"
+
+# Allow TCP with TLS from platform
+gcloud compute firewall-rules create allow-platform-tls \
+  --allow tcp:8443 \
+  --source-ranges 0.0.0.0/0 \
+  --description "Allow TCP with TLS from platform"
+```
+
+### Azure
+
+Configure Network Security Groups:
+
+```json
+{
+  "securityRules": [
+    {
+      "name": "AllowGatewaySSH",
+      "properties": {
+        "protocol": "Tcp",
+        "sourcePortRange": "*",
+        "destinationPortRange": "2222",
+        "sourceAddressPrefix": "*",
+        "destinationAddressPrefix": "*",
+        "access": "Allow",
+        "priority": 100,
+        "direction": "Inbound"
+      }
+    },
+    {
+      "name": "AllowPlatformTLS",
+      "properties": {
+        "protocol": "Tcp",
+        "sourcePortRange": "*",
+        "destinationPortRange": "8443",
+        "sourceAddressPrefix": "*",
+        "destinationAddressPrefix": "*",
+        "access": "Allow",
+        "priority": 110,
+        "direction": "Inbound"
+      }
+    }
+  ]
+}
+```
+
+## Protocol Details
+
+### SSH over TCP
+
+The gateway uses SSH reverse tunnels for primary communication:
+
+- **Port 2222**: SSH connection from gateways to relay servers
+- **Built-in features**: Automatic reconnection, certificate-based authentication, encrypted tunneling
+- **Encryption**: SSH with certificate-based authentication and key exchange
+
+### TCP Connection Handling
+
+SSH connections over TCP are stateful and handled seamlessly by all modern firewalls:
+
+- **Established connections** are automatically tracked
+- **Return traffic** is allowed for established outbound connections
+- **No special configuration** needed for connection tracking
+- **Standard SSH protocol** that enterprise firewalls handle well
+
+## Performance Considerations
+
+### Network Optimization
```

Comment:
i don't think this is going to be too helpful. people already know to deploy relay close to their resource. we also mention it in the beginning